package.json

```diff
@@ -17,12 +18,15 @@
     "#preflights": "./src/_preflights/index.js",
     "#postprocess": "./src/postprocessor.js"
   },
+  "files": [
+    "dist"
```

Comment:
I'm pretty sure this would cause some of the specified `exports` to fail.

Reply:
Good point. Do we want everything from the exports entry to be published to NPM? Right now all repo contents are available there, which doesn't sound necessary. What do you think?

Reply:
I think we can use an `.npmignore` file to remove our test files - otherwise I think everything else is used.

Reply:
I see you tried adding a rule `!dist/` to `. npmignore`, which sounds to me as if you wanted to publish just the dist folder after all. That rule didn't work so we published everything. Do we indeed want to just exclude the tests folder? Just trying to connect the dots :)

Reply:
Instead of adding more to `.npmignore` I was thinking to set `src`, `dist` and `dev.js` as files published to NPM. Wdyt? [f415da5](https://github.com/warp-ds/drive/pull/68/commits/f415da56de51dfb431c3a46428ef6852841dabcc)

Reply:
That works!